contrib/docker/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM ubuntu:14.04
```

Comment:
> I had some trouble getting Node to run on debian

I'm using Debian with the nodesource dpkg server with no issues. But well, it's not with docker.